omnibus/files/private-chef-cookbooks/private-chef/recipes/opscode-solr4.rb

```diff
@@ -90,6 +93,15 @@
   notifies :restart, 'runit_service[opscode-solr4]' if is_data_master?
 end
 
+template File.join(solr_jetty_dir, "contexts", "solr-jetty-context.xml") do
+  owner OmnibusHelper.new(node).ownership['owner']
+  group OmnibusHelper.new(node).ownership['group']
+  mode "0644"
+  source "solr4/solr-jetty-context.xml.erb"
+  notifies :restart, 'runit_service[opscode-solr4]' if is_data_master?
+end
```

Comment:
Why is this a template and not a file resource? Nothing looks configurable here.